src/types.js

```diff
@@ -1,12 +1,72 @@
 export const TYPES = {
   tos: {
```

Comment:
Maybe we should expand acronyms 🤔

Reply:
Yep 👍 , I was thinking the same the last week.